lyndapham/step/portfolio/src/main/webapp/essays.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>My Essays</title>
+    <link rel="stylesheet" href="style.css">
+    <!-- <script src="script.js"></script> -->
+  </head>
+  <body>
+        <main>
+            <h1>Lynda's Thoughts</h1>
+            <div class="content">
+                <div class="pdf">
+                    <h3>Driving</h3>
+                    <object width="360" height="270" data="pdfs/Driving.pdf"></object>
+                </div>
+
+                <div class="pdf">
+                    <h3>Pink</h3>
+                    <object width="360" height="270" data="pdfs/Pink.pdf"></object>
+                </div>
+
+                <div class="pdf">
+                    <h3>Short Stories</h3>
+                    <object width="360" height="270" data="pdfs/Vocab.pdf"></object>
+                </div>
+            </div>
+            <p><a href="index.html">Return to the homepage!</a></p>
+        </main>
+  </body>
+</html>
```

Comment:
Add a new line at the end of the file. Looks like github also complains about this as well.

Reply:
Where can I see where github gives feedback?